### ISSUE TITLE
i.mount_at_root = mount potential problem?

### ISSUE THREAD

**Issue by andreamoro** [iwarner/middleman-blog-test#1]
Provided that I'm playing around with your integration over mine (yours seems to work with the code provided), I think that having the ```i.mount_at_root = mount``` always set to the same language will create in the long term run.

to work fine mount_at_root should be set to false, so each localised version can live on its own "workspace"

**Comment by iwarner** [comment]
@andreamoro Hiya I have no idea what this means :) Whats the problem you are describing and solution you want.

**Comment by andreamoro** [comment]
Fair point. I have not given proper details you're right. What if we catch-up on Skype so to nail down all these issues in the community interest to solve the "blog" i18n integration?
